### PR TITLE
sponsors page made responsive

### PR DESCRIPTION
**Pull request by sambhavnrana**

### CODE REVIEW

**Comment by psantosh16** [comment]
Don't make duplicate PR's. And delete which ever is not needed

**Review by psantosh16 — CHANGES_REQUESTED** [review]
Kindly delete previous unmergerd PRs SambhavJii

File: src/components/sponsors/sponsors.jsx
Comment (psantosh16) [inline src/components/sponsors/sponsors.jsx:1]: Use NextJs  <Image> component

Comment (psantosh16) [inline src/components/sponsors/sponsors.jsx:1]: Use Image from NEXTJS instead of img

👍 Nice work in maintaining clean code

File: src/app/page.jsx
Comment (psantosh16) [inline src/app/page.jsx:1]: Import SponsorsPage from sponsors instead of content

**Review by psantosh16 — CHANGES_REQUESTED** [review]
Changes are needed

**Comment by sambhavnrana** [comment]
> Don't make duplicate PR's. And delete which ever is not needed

Okay

**Review by psantosh16 — APPROVED** [review]

**Review by psantosh16 — APPROVED** [review]

**Review by psantosh16 — APPROVED** [review]

**Review by psantosh16 — APPROVED** [review]

**Pull request merged**